assets/js/blocks/legacy-template/editor.scss

```diff
@@ -14,7 +14,7 @@
 	}
 }
 
-.wp-block-woocommerce-legacy-template__placeholder-image {
+.editor-styles-wrapper .wp-block-woocommerce-legacy-template__placeholder-image {
```

Comment:
If we're adding this class just for increased specificity, can we use `wp-block-woocommerce-legacy-template__placeholder` instead of `.editor-styles-wrapper` here? This way we're using only classes we control within the scope of our plugin.